binascii

```diff
@@ -0,0 +1,285 @@
+%!PS-Adobe-3.0
```

Comment:
Please remove this file and the file `python_can`, `socket`, `threading` from your PR. 
Please undo the modification to `isotpscanner.py`.

Reply:
Deleted.
Do you mean actual changes to `ìsotpscanner.py`?
I am probably missing something, but if I compare it to your last version of the file, I don't see any alterations.

Reply:
You changed the file permissions, I think

Reply:
Yes, you are right!
I changed it back to 644